Styling issues to be fixed in configurator lib (#17103)
SAP/spartacus: docs/migration/6_0/styling.md

#### ConfiguratorAttributeHeaderComponent

- Set `margin-block-end` to `0px` instead of `17px`
- Added `.cx-action-link { font-size: 14px; min-height: 0px;} `to `.cx-conflict-msg `
- Added `padding-block-end: 5px; padding-block-start: 5px; display: block;` to `img`

#### ConfiguratorAttributeReadOnlyComponent

- `%cx-configurator-attribute-read-only` now only contains `@include cx-configurator-attribute-type();`. This mixin already has all
  the needed styling settings

#### ConfigureCartEntryComponent

> - Styling file is obsolet and will be removed because `link cx-action-link` classes are in used

Reviewer: ```suggestion
- Styling file was obsolete and has been removed because `link cx-action-link` classes are used
```

Author: Done!

- `margin-inline-start` changed to `-15px` instead of `-20px`
- `margin-inline-end` changed to `-15px` instead of `-20px`
- Added `padding-inline-start: 0px;` to `.cx-title`

#### ConfiguratorExitButtonComponent

> - Styling file is obsolet and will be removed 

Reviewer: ```suggestion
- Styling file was obsolete and has been removed 
```

Author: Done!
